src/store/storeConfiguration.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, applyMiddleware } from 'redux';
+import reduxImmutableStateInvariant from 'redux-immutable-state-invariant';
+import thunk from 'redux-thunk';
+import { logger } from 'redux-logger';
+import rootReducer from '../reducers';
+
+// initializes the store with a state
+export default function configureStore(initialState) {
```

Comment:
Try and use the `es6` formats while writing functions.

Reply:
I have changed that